fem/bilininteg.hpp

```diff
@@ -2602,6 +2608,31 @@ class VectorCurlCurlIntegrator: public BilinearFormIntegrator
                                    const Vector &elfun);
 };
 
+/** Class for integrating the bilinear form a(u,v) := (Q curl u, v) where Q is a
+    scalar coefficient, and v is a vector with components v_i in the L2 or H1 space.
+    u can be in H(curl) (2D or 3D) or it can be a scalar H1.
+    Note: If u is scalar H1 then curl u = [0 1; -1 0] grad u */
```

Comment:
Doesn't `MixedScalarCurlIntegrator` already support some cases of this integrator, while this integrator is more general? Does this integrator make `MixedScalarCurlIntegrator` obsolete? In any case, it would be helpful to comment here what relation, if any, there is between the two integrators.

Reply:
done